proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -247,7 +247,9 @@ message MsgUnbondValidator {
   option (cosmos.msg.v1.signer) = "validator_address";
   option (amino.name)           = "cosmos-sdk/MsgUnbondValidator";
 
-  string validator_address = 1 [(gogoproto.moretags) = "yaml:\"address\""];
+  // the address must be provided in bech32 notation e.g. cosmos1apsdadpks9i09idj87dah
+  // providing valoper or valcons address will result in an error
```

Comment:
I'm not that familiar, but aren't the `comsosvalcons` and `cosmosvaloper` addresses also in Bech32? If yes, the "must be provided in bech32 notation" might be slightly confusing.
Also, seems slightly peculiar that the base account address is needed. I guess this stems from the fact that the address stems from the `--from` argument and it is the validator that executes this command?

Reply:
It's difficult to debug and we've spent more time on it than it diserves.

The problem is solved by not treating the `--from` address as `cosmosvaloper` and instead using `cosmos`.

Something goes wrong on sign/broadcast and it's terribly difficult to debug those because it passes in all types of tests and not on the app. This was broken on both `gaiad` and the `simd`.

The addresses types are just a display format, the underlying address bytes are the same accross for `cosmos` and `cosmosvaloper`.

Reply:
Clarified in https://github.com/cosmos/cosmos-sdk/pull/21138/commits/a7e513f887f4b069590681bb99eb871a5c295763